### PR TITLE
Use wgs84 transform for royal exhibition

### PR DESCRIPTION
**Pull request by loshjawrence**
This overrides the transform for royal exhibition until the actual tileset.json transform is fixed, after which, the override can be removed. 

Also includes some fixes to bounding-volume to use real oriented-bound-box which is part of the way to using real distance checks.

### CODE REVIEW

File: modules/3d-tiles/src/tileset/helpers/bounding-volume.js [hunk 1]
```diff
@@ -72,21 +65,30 @@ export function createBoundingVolume(boundingVolumeHeader, transform, result) {
...
-
-  const center = new Vector3(box[0], box[1], box[2], scratchCenter);
-  const halfAxes = new Matrix3(); // Matrix3.fromArray(box, 3, scratchHalfAxes);
-
-  // Find the transformed center and halfAxes
-  center = Matrix4.multiplyByPoint(transform, center, center);
-  const rotationScale = Matrix4.getRotation(transform, scratchMatrix);
-  halfAxes = Matrix3.multiply(rotationScale, halfAxes, halfAxes);
+  const center = new Vector3(box[0], box[1], box[2]);
+  let halfAxes = new Matrix3(box.slice(3, box.length));
```
Comment (ibgreen) [inline modules/3d-tiles/src/tileset/helpers/bounding-volume.js]: You never use this value

File: examples/3d-tiles/package.json [hunk 2]
```diff
@@ -13,10 +13,10 @@
     "@loaders.gl/3d-tiles": "^1.0.3",
     "@loaders.gl/core": "^1.0.3",
     "@luma.gl/addons": "^7.1.0",
-    "math.gl": "^3.0.0-alpha.3",
-    "@math.gl/geospatial": "^3.0.0-alpha.3",
-    "@math.gl/culling": "^3.0.0-alpha.3",
+    "@math.gl/culling": "^3.0.0-alpha.5",
```
Comment (ibgreen) [inline examples/3d-tiles/package.json]: Nit: I think xintong published an alpha.6.

File: examples/3d-tiles/tile-3d-layer.js [hunk 3]
```diff
@@ -50,6 +50,29 @@ export default class Tile3DLayer extends CompositeLayer {
     let tileset3d = null;
     if (tilesetUrl) {
       const tilesetJson = await load(tilesetUrl);
+
+      if (tilesetUrl.includes('RoyalExhibitionBuilding')) {
```
Comment (ibgreen) [inline examples/3d-tiles/tile-3d-layer.js]: You should be able to cut this now.

File: modules/3d-tiles/src/tileset/helpers/bounding-volume.js (hunk 1 above)
Comment (loshjawrence) [reply modules/3d-tiles/src/tileset/helpers/bounding-volume.js]: Not sure what you mean

**Comment by ibgreen** [comment]
@loshjawrence I can land this as soon as you have rebased to master

File: modules/3d-tiles/src/tileset/helpers/bounding-volume.js (hunk 1 above)
Comment (ibgreen) [reply modules/3d-tiles/src/tileset/helpers/bounding-volume.js]: You are assigning `halfAxes` on line 69, and overwriting it on line 73, as far as I can tell, you don't use the value in between.

**Comment by loshjawrence** [comment]
@ibgreen 
Having some trouble compliing after the merge something about babel/polyfill:
image

**Comment by ibgreen** [comment]
>  about babel/polyfill:

after rebasing, it is a good idea to run yarn in the example (and ideally also yarn bootstrap in root).

**Comment by loshjawrence** [comment]
the bootstrap did it, thanks.

File: modules/3d-tiles/src/tileset/helpers/bounding-volume.js (hunk 1 above)
Comment (loshjawrence) [reply modules/3d-tiles/src/tileset/helpers/bounding-volume.js]: there's a multiplyRight on 82 that uses it

Comment (loshjawrence) [reply modules/3d-tiles/src/tileset/helpers/bounding-volume.js]: There's a multiplyRight on 82 that uses it.

**Comment by loshjawrence** [comment]
@ibgreen I think this is good to go, unless the halfAxes multiplyRight usage on 82 could be made more readable or something.

File: modules/3d-tiles/src/tileset/helpers/bounding-volume.js (hunk 1 above)
Comment (ibgreen) [reply modules/3d-tiles/src/tileset/helpers/bounding-volume.js]: Nevermind... I looks to me like it uses the value you assign on line 72. The value you initialize with on line 68 is overwritten and never used.

**Pull request merged**